UI/data/locale/en-US.ini

```diff
@@ -266,6 +266,8 @@ Updater.Running.Title="Program currently active"
 Updater.Running.Text="Outputs are currently active, please shut down any active outputs before attempting to update"
 Updater.NoUpdatesAvailable.Title="No updates available"
 Updater.NoUpdatesAvailable.Text="No updates are currently available"
+Updater.BranchNotFound.Title="Update channel removed"
```

Comment:
Dialog titles should be Title Case.

Reply:
I changed it, although it should be noted that none of the others are Title Case (except the integrity check ones I added earlier this year).

Reply:
Yeah it's something I hate about our dialogs. One day I'm gonna go through them all...